strict-checked-vars/test/Test/Control/Concurrent/Class/MonadMVar/Strict/Checked.hs

```diff
@@ -43,3 +42,10 @@ prop_invariantShouldFail :: MonadMVar m => PropertyM m ()
 prop_invariantShouldFail = run $ do
     v <- newMVarWithInvariant invPositiveInt 0
     modifyMVar_ v (\x -> pure $ x - 1)
+
+shouldFail :: Property -> Property
```

Comment:
I'd suggest a less-biased name, since `shouldFail = id` is a bit non-sensical.

Perhaps `cppToggle` or so?